pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -4353,8 +4442,16 @@ def PumpCostRule(b, l1, l2):
         the cost of pumps.
         """
         # In the co_optimize method the objective should include the cost of pumps. So,
-        # delete the original objective to add a modified objective in this method.
-        del model.objective
+        # delete the original objectives to add a modified objective in this method.
+        del model.objective_Cost
+        del model.objective_Reuse
+        if model.do_subsurface_risk_calcs:
+            del model.objective_SubsurfaceRisk
```

Comment:
Similarly, here we delete the SubsurfaceRisk Objective not sure why.

Reply:
Good point - instead of deleting, I changed the code to deactivate the original objective functions.